etc/systemd/system/primary.duffy.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+After=network.target
+Requires=worker.duffy.service
+
+[Service]
+User=duffy
+Group=duffy
+ExecStart=/venv/bin/duffy -c /etc/duffy-example-config.yaml serve
+Restart=on-failure
+RestartSec=60
+
+[Install]
+WantedBy=default.target
+RequiredBy=legacy.duffy.service
```

Comment:
This is missing a newline at the end of the file (I wonder why the other files don't).